db/buckets.go

```diff
@@ -29,7 +29,7 @@ const (
 	ContractDeploymentHeight
 	L1Height
 	SchemaVersion
-	Pending
+	Unused
```

Comment:
do you think it's worth adding a comment i.e. it was Pending previously

Reply:
done